### ISSUE TITLE
Startup Crash on Fabric 1.19.2 (V1.31)

### ISSUE THREAD

**Issue by Kaede4Life** [Womp6/Womp-s-Crayfish-Mod-Issues#1]
Hi, I am experiencing a startup crash with the latest version of the mod. Here is the crash log:

https://gist.github.com/Kaede4Life/47d30b1adff2a78b414701140d8c9061

**Comment by Womp6** [comment]
Hello, thanks for bringing this issue to my attention.  I test the versions on 1.19 (which this is not an issue on for some reason) because I mainly play on it as not all the mods I use are updated to 1.19.2 yet but for some reason I didn't see this when I tested the mod on 1.19.2 before releasing it.  Although I remember I was in a hurry and probably ran into this issue but forgot later when I released it for 1.19.2.  I am sorry for the inconvenience and this issue will hopefully be resolved and uploaded in a new version tomorrow.  I will comment again to let you know when it is resolved.  (I will also test on 1.19.2 from now on after this)
